packages/react-sdk/bin/generate-arg-types.ts

```diff
@@ -33,11 +34,22 @@ if (componentFiles.length === 0) {
 // Create a parser with using your typescript config
 const tsConfigParser = withCustomConfig(tsConfigPath, options);
 
+const schema = z.record(
```

Comment:
convention for zod schemas is always capital letter, ideally call it a proper name, I think this is `ArgTypes`